README.md

```diff
@@ -46,7 +48,16 @@ pipe([1, 2, 3], stream)
 
 ## API
 
-### `const muxer = new Mplex([options])`
+### `const factory = new Mplex([options])`
+
+Creates a factory that can be used to create new muxers.
+
+`options` is an optional `Object` that may have the following properties:
+
+* `maxMsgSize` - a number that defines how large mplex data messages can be in bytes, if messages are larger than this they will be sent as multiple messages (default: 1 << 20)
```

Comment:
minor thing, i think describing the default as a power of 2 is more readable (eg: 2 ** 20)
or just writing the number